docker/include/bin/start_faux

```diff
@@ -87,7 +87,23 @@ if [ -n "${options[xdhcp]}" ]; then
     fi
     ip addr show $intf_name
 
-    (while true; do ping -c 1 10.20.0.1; sleep 5; done) &
+    if [ -n "${options[opendns]}" ]; then
+        route add -net 172.17.0.0/16 $intf_name  # default docker ip range.
+        dhcp_dns=8.8.8.8
+        echo nameserver ${dhcp_dns#DNS=} > /etc/resolv.conf
+        # There is no deterministic way to find out the default gateway ip where running more than 1 faux device.
+        (
+            while true; do
+                if [ `route -n | grep UG | wc -l` -eq '1' ]; then 
+                    timeout 1 ping -c 1 `route -n | grep UG ` && continue
+                fi 
+                for IP in {1..12}
+                do
+                    timeout 1 ping -c 1 172.17.0.$IP  && route add default gw 172.17.0.$IP $intf_name && break
+                done
+            done
+        ) &
```

Comment:
does this handle the case where the GW address changes?   I think the address can keep changing... ?

Reply:
Yes so if ping to gateway stops responding, it will go through the bruteforce again

Reply:
does it keep adding default routes, or do they get removed somehow?

Reply:
I forgot to remove them but they still worked fine. I think system will block off the dead one.

Reply:
Most likely just takes the most recent since I don't think there's any way for the system to know which is dead... but either way as long as it works.